apps/tutorial_servo_control/src/header.h

```diff
@@ -0,0 +1,51 @@
+#ifndef TUTORIAL_SERVO_H
+#define TUTORIAL_SERVO_H
+
+#include "mbed.h"
```

Comment:
hmm it still looks like youre not seperating declaration and implementation. we could have a call sometime to chat abt what i mean by this. lmk if you want that. otherwise read this link
http://www.math.uaa.alaska.edu/~afkjm/csce211/handouts/SeparateCompilation.pdf